package.json

```diff
@@ -29,17 +29,17 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-adb": "2.0.0-beta14",
+    "appium-adb": "2.0.0-beta15",
     "appium-android-bootstrap": "^2.1.5",
     "appium-android-ime": "^2.0.0",
     "appium-base-driver": "^0.4.1",
     "appium-chromedriver": "^2.3.0",
-    "appium-express": "^0.0.5",
+    "appium-express": "^0.0.6",
     "appium-logger": "^1.1.7",
     "appium-support": "2.0.0-beta21",
     "appium-unlock": "0.0.1",
     "asyncbox": "^2.0.4",
-    "babel-runtime": "=5.5.5",
+    "babel-runtime": "5.8.20",
```

Comment:
Is this the one we want to use? If so, we also need `=` here or we sometimes run into problems with babel